requirements.txt

```diff
@@ -1,8 +1,13 @@
-streamlit
-streamlit-plotly-events
-streamlit-aggrid
-pandas
-numpy
-plotly
---index-url https://pypi.cs.uni-tuebingen.de/simple/
-pyopenms-nightly
\ No newline at end of file
+# the requirements.txt file is intended for deployment on streamlit cloud
```

Comment:
If we are going to comment out here, can we have a docker file only with pyopenms? I tried to build the docker file, and it is taking way more time than I expected it to be due to the OpenMS + third-party. The initial way (using this requirements.txt file) was much faster and easy...

Reply:
I understand. One question: Is this file commented out because of the docker build or git action? Otherwise, how about we just provide this file as a "simpler version" and the docker file as a "developer version"?

Reply:
maybe we could also add one simpler example that just uses the requirements file in the docker file

Reply:
Sounds great to me!